docs/developer_guide/contributing.md

```diff
@@ -0,0 +1,100 @@
+# Contributing
+
+Everybody is welcome to contribute, and we value everybody’s contribution. Code is thus not the only
+way to help the community. Answering questions, helping others, reaching out and improving the
+documentation are immensely valuable contributions as well.
+
+It also helps us if you spread the word: reference the library from blog posts on the awesome
```

Comment:
nit: maybe it needs to be changed to `reference the library in blog posts..`?

Reply:
done.